include/nbl/builtin/hlsl/tgmath/impl.hlsl

```diff
@@ -429,6 +451,91 @@ struct erfInv_helper<FloatingPoint NBL_PARTIAL_REQ_BOT(concepts::FloatingPointSc
 	}
 };
 
+template<>
+struct erfInv_helper<float64_t>
+{
+	static float64_t __call(NBL_CONST_REF_ARG(float64_t) _x)
+	{
+		float64_t x = clamp<float64_t>(_x, NBL_FP64_LITERAL(-0.99999), NBL_FP64_LITERAL(0.99999));
+
+		float64_t w = float64_t(-log_helper<float32_t>::__call((float32_t(NBL_FP64_LITERAL(1.0)) - x) * float32_t(NBL_FP64_LITERAL(1.0)) + x));
```

Comment:
use `float64_t` in the exp, and then comment out (or otherwise disable) the entire partial spec

Reply:
also check the license of the other implementations of our `erf` and `erfInv` functions and make sure they don't come from GLIBC or other GPL thing, and if they do find different library which has them under a different license AND MAKE SURE TO WRITE A COMMENT STATING WHERE THE CODE CAME FROM

Reply:
The current ones don't seem to have a license attached. The `erf` approximations came from the formula in A&S "Handbook of Mathematical Functions". `erfInv` is from GPU Computing Gems.

I could re-add an `erf<float64_t>` implementation from MUSL `libm` which uses apache license like us. But it's still for HLSL only so not that useful.

Reply:
then write down for the float64_t

Reply:
also note the book, usually books are sane with their code snippets and don't license them weirdly (thats the whole point of selling them)

Reply:
@keptsecret can you please put in the comments where the code comes from!?